fuzzers/lafintel/builder.Dockerfile

```diff
@@ -0,0 +1,56 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG parent_image=gcr.io/fuzzbench/base-builder
+FROM $parent_image
+
+# Need Clang/LLVM 3.8.
+RUN apt-get update -y && \
+    apt-get -y install llvm-3.8 \
+    clang-3.8 \
+    libstdc++-5-dev \
+    wget
+
+# Download AFL and compile using default compiler.
+# We need afl-2.26b
+RUN wget https://lcamtuf.coredump.cx/afl/releases/afl-2.26b.tgz -O /afl-2.26b.tgz && \
+    tar xvzf /afl-2.26b.tgz -C / && \
+    mv /afl-2.26b /afl && \
+    cd /afl && \
+    AFL_NO_X86=1 make
+
+# Set the env variables for LLVM passes and test units.
+ENV CC=clang-3.8
+ENV CXX=clang++-3.8
+ENV LLVM_CONFIG=llvm-config-3.8
+
+# Build the LLVM passes with the LAF-INTEL patches, using Clang 3.8.
+RUN cd /afl/llvm_mode && \
+    wget https://gitlab.com/laf-intel/laf-llvm-pass/raw/master/src/afl.patch && \
+    patch -p0 < afl.patch && \
+    wget https://gitlab.com/laf-intel/laf-llvm-pass/raw/master/src/compare-transform-pass.so.cc && \
+    wget https://gitlab.com/laf-intel/laf-llvm-pass/raw/master/src/split-compares-pass.so.cc && \
+    wget https://gitlab.com/laf-intel/laf-llvm-pass/raw/master/src/split-switches-pass.so.cc && \
+    sed -i 's/errs()/outs()/g' split-switches-pass.so.cc && \
+    sed -i 's/errs()/outs()/g' split-compares-pass.so.cc && \
+    sed -i 's/errs()/outs()/g' compare-transform-pass.so.cc && \
```

Comment:
just wondering, what's wrong with printing to stderr?

Reply:
@lmrs2 might know this, I think it's his code.

Reply:
good question. One benchmark failed (json, if memory serves). I think the compilation tool interpreted message in stderr as an error. Using outs() fixed the issue

Reply:
I think setting the env var `AFL_QUIET=1` might fix this, I think I encountered this problem when compiling a benchmark with AFLSmart

Reply:
(just realized this pass is from outside AFL so it probably doesn't support this).

Reply:
AFL_QUIET is not supported in the old AFL version. But you're also right, those passes would not know about it either.